### PR TITLE
Maria howtos web modeler

### PR DESCRIPTION
**Pull request by MariaShaposhnikova**
add new how-to

### CODE REVIEW

File: content/howto/web-modeler/index.md [hunk 1]
```diff
@@ -5,4 +5,6 @@ title: "Web Modeler"
 ## Documents in This Category
 
 * [How to Sync the Web Modeler with the Desktop Modeler](syncing-webmodeler-desktop)
+* [How to Configure an Exclusive Split in the Web Modeler Step 1: Adding Entities and Attributes to the Domain Model, and Configuring a Microflow](webmodeler-how-to-microflows-exclsplit)
```
Comment (Adam-Dupaski) [inline content/howto/web-modeler/index.md]: The titles need to be updated here to reflect the actual titles without "Web Modeler."

File: content/howto/web-modeler/webmodeler-how-to-microflows-exclsplit-p2.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+To start this tutorial, make sure you have completed the following prerequisites:
+
+* [Configure an Exclusive Split Step 1](webmodeler-how-to-microflows-exclsplit)
+
+## 3 Embedding the Microflow to Pages   
+
+After microflows are created, you can add them to pages to run them in your app. 
+
+### 3.1 Embedding the Microflow Having the Exclusive Split with Attribute of the Enumeration Type {#embedding-exclusive-split-enumeration}
```
Comment (Adam-Dupaski) [inline content/howto/web-modeler/webmodeler-how-to-microflows-exclsplit-p2.md]: This heading is probably too long. "Having the" should be "with an"

File: content/howto/web-modeler/webmodeler-how-to-microflows-exclsplit-p2.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+After microflows are created, you can add them to pages to run them in your app. 
+
+### 3.1 Embedding the Microflow Having the Exclusive Split with Attribute of the Enumeration Type {#embedding-exclusive-split-enumeration} 
+
+To embed the microflow with an exclusive split (the attribute of the enumeration type) to pages, do the following:
+
+1. Сreate a page for the existing customers details and name it *Customer_details*. For more information on creating pages, see section [3.2 Creating a new Page](../../refguide/web-modeler/page-editor-wm).
+2.  In **Toolbox**>**Widgets** >**Data Containers**, find **Data View**.
+
+   ![](attachments/webmodeler-how-to-microflows-exclsplit/wm-data-view.png)
```
Comment (Adam-Dupaski) [inline content/howto/web-modeler/webmodeler-how-to-microflows-exclsplit-p2.md]: Will probably need an extra space to make attachment code parallel with numbered item above.

File: content/howto/web-modeler/webmodeler-how-to-microflows-exclsplit-p2.md [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+
+3. Drag and drop **Data View** to the page.
+4. In the **Properties** tab for the **DATA VIEW**, do the following: 
+   a. Set **Data Source** to **Context.**
```
Comment (Adam-Dupaski) [inline content/howto/web-modeler/webmodeler-how-to-microflows-exclsplit-p2.md]: Will need extra spaces for attachment. Will also need `<br />` for the a-b sub-list. Check the preview to see the problems with this formatting throughout.

File: content/howto/web-modeler/webmodeler-how-to-microflows-exclsplit-p2.md [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+
+Congratulations! Now when the user clicks **Details**, the form for the corresponding customer grade will be opened. 
+
+You can now publish and view your app. For more information, see [How to Publish and View Your App](../tutorials/start-with-a-blank-app-3-publish-and-view-your-app).
+
+
+
+### 3.2 Embedding the Microflow Having the Exclusive Split with the Attribute of the Boolean Type 
+
+To embed the microflow with an exclusive split (the attribute of the boolean type), do the following:
```
Comment (Adam-Dupaski) [inline content/howto/web-modeler/webmodeler-how-to-microflows-exclsplit-p2.md]: "Boolean" should be capitalized, as it's named after a person. Other data types should not.

File: content/howto/web-modeler/webmodeler-how-to-microflows-exclsplit.md [hunk 6]
```diff
@@ -0,0 +1,225 @@
+---
+title: "Configure an Exclusive Split Step 1: Add Entities and Attributes to the Domain Model, and Configure a Microflow"
```
Comment (Adam-Dupaski) [inline content/howto/web-modeler/webmodeler-how-to-microflows-exclsplit.md]: Title is probably too long, consider replacing "Add Entities and Attributes to the Domain Model" with "Build the Domain Model". The titles/headings do not need to reflect exactly everything the user will do in the how-to.

File: content/howto/web-modeler/webmodeler-how-to-microflows-exclsplit.md [hunk 7]
```diff
@@ -0,0 +1,225 @@
...
+category: "Web Modeler"
+description: "This how to describes the process of configuring an exclusive split in the Mendix Web Modeler."
```
Comment (Adam-Dupaski) [inline content/howto/web-modeler/webmodeler-how-to-microflows-exclsplit.md]: Add a hyphen: "how-to"

File: content/howto/web-modeler/webmodeler-how-to-microflows-exclsplit.md [hunk 8]
```diff
@@ -0,0 +1,225 @@
...
+tags: ["web modeler", "microflows", "exclusive split", "domain model"]
+---
+
+## 1 Introduction
+
+This how-to explains how you can configure an exclusive split in the Microflow editor of the Web Modeler.
```
Comment (Adam-Dupaski) [inline content/howto/web-modeler/webmodeler-how-to-microflows-exclsplit.md]: We do not capitalize editors: "microflow editor."

File: content/howto/web-modeler/webmodeler-how-to-microflows-exclsplit.md [hunk 9]
```diff
@@ -0,0 +1,225 @@
...
+
+**This how-to will teach you how to do the following:**
+
+* Add entities and attributes necessary to configure the exclusive split
+* Configure the exclusive split with the boolean type of parameter or attribute
+* Configure the exclusive split with the enumeration type of parameter or attribute
+
+This how-to describes the following use case: 
+
+You have an online shopping app. You will create a page where you can manage your customer's details, i.e, name, grade, and the customer's status (active or blocked).
```
Comment (Adam-Dupaski) [inline content/howto/web-modeler/webmodeler-how-to-microflows-exclsplit.md]: We do not use "i.e.," - please see the style guide.

File: content/howto/web-modeler/webmodeler-how-to-microflows-exclsplit.md [hunk 10]
```diff
@@ -0,0 +1,225 @@
...
+In this example we will create a microflow and configure the exclusive split when you want to open different order forms depending on the customer grade. 
+
+This use case will require an exclusive split with attribute of the enumeration type (list of predefined values). For more information on types of attributes, see [Attribute Types](../../refguide/web-modeler/domain-models-attributes-wm). 
+
+### 3.1 Adding an Entity and an Attribute to the Domain Model 
+
+The app will open the corresponding page depending on the grade of a customer, for this we need to create a new entity and a new attribute first. To create the new entity and attribute, do the following:
+
+1. Open your [domain model](../../refguide/web-modeler/domain-models-wm).
+2. Create entity *Customer*. For more information on how to create the entity, see section [3 Adding New Entities](../../refguide/web-modeler/domain-models-wm).
```
Comment (Adam-Dupaski) [inline content/howto/web-modeler/webmodeler-how-to-microflows-exclsplit.md]: If you're linking to a section in another doc, you need to name the other doc as well, or else the reader will assume that section is in the same doc that they are currently reading. So: "see section [3 Adding New Entities](../../refguide/web-modeler/domain-models-wm) in *Domain Models Overview in the Web Modeler*."

File: content/howto/web-modeler/webmodeler-how-to-microflows-exclsplit.md [hunk 11]
```diff
@@ -0,0 +1,225 @@
...
+3. For the **Customer** entity, create attribute (for more information on how to create the attribute, see section [4 Adding New Attributes](../../refguide/web-modeler/domain-models-wm)) and do the following:<br />
+  a. Set the attribute **Name** to *Grade*.<br />
+  b. Set the [**Type**](../../refguide/web-modeler/domain-models-attributes-wm) to **Enumeration**.<br />
+  c. Click **Select enumeration** to create a new enumeration.<br />
+
+  ![](attachments/webmodeler-how-to-microflows-exclsplit/wm-create-new-attribute-enumeration.png)
```
Comment (Adam-Dupaski) [inline content/howto/web-modeler/webmodeler-how-to-microflows-exclsplit.md]: Need `<br />` after the image for the list to be consistent. Please see the preview for the problems with this list.

File: content/howto/web-modeler/webmodeler-how-to-microflows-exclsplit.md [hunk 12]
```diff
@@ -0,0 +1,225 @@
...
+   a. Select **Edit** in the **(not set)** field.<br />
+
+   ![](attachments/webmodeler-how-to-microflows-exclsplit/wm-setting-cases-for-excl-split.png)
+   b. Set **Bronze** in the **Select Value** drop-down menu.<br />
+   c. Click the **Go back** icon to return to the exclusive split properties.<br />
+
+   ![](attachments/webmodeler-how-to-microflows-exclsplit/wm-excl-split-go-back.png)
+   d. Click **Add New Case** in the **Cases** section.<br />
+   e. Repeat steps b-d to add all possible cases: **Silver**, **Gold** and **Empty** (a case when the customer's grade is not set). 
+
```
Comment (Adam-Dupaski) [inline content/howto/web-modeler/webmodeler-how-to-microflows-exclsplit.md]: Problems throughout these lists with the formatting and consistency. Please fix according to the preview. Also, consider using fewer screenshots, they are not all necessary - we don't need a screenshot for each list item, only where the user might have trouble and the text-instruction needs to be supplemented with an image for clarity.

File: content/howto/web-modeler/webmodeler-how-to-microflows-exclsplit.md [hunk 13]
```diff
@@ -0,0 +1,225 @@
...
+12. In the **Properties** tab for the **Show message** activity, do the following:
+    a. Select **Error** as the message type.<br />
+    b. Fill out the **Template** that will be shown to users when this message pops up (in our example: Please select the customer grade first).<br />
+    c. Leave the **Blocking** property for the message enabled, which prevents the user continue work until the pop-up window is closed.  
+
+    ![](attachments/webmodeler-how-to-microflows-exclsplit/wm-empty-customer-grade-message.png)
+
+Congratulations! We have now created the microflow that will open different order forms for customers with different grades, or show an error message when the customer has no grade.
+
+If you want to test your microflow by adding it to the pages, see [Configure an Exclusive Split in the Web Modeler Step 2: Embedding the Microflow to Your App](webmodeler-how-to-microflows-exclsplit-p2).
```
Comment (Adam-Dupaski) [inline content/howto/web-modeler/webmodeler-how-to-microflows-exclsplit.md]: Make titles of these how-to's consistent throughout.

File: content/howto/web-modeler/webmodeler-how-to-microflows-exclsplit-p2.md (hunk 2 above)
Comment (MariaShaposhnikova) [reply content/howto/web-modeler/webmodeler-how-to-microflows-exclsplit-p2.md]: I can't come up with a shorter one that underlines the attribute type is enumeration and differentiates 3.1. from 3.2. Any suggestions?

**Pull request merged**